AdminWebsite/AdminWebsite/ClientApp/src/app/popups/wait-popup/wait-popup.component.html

```diff
@@ -1,7 +1,6 @@
 <div id="waitPopup" class="vh-popup-overlay">
   <div class="vh-popup vh-popup-xsmall">
-    <h1 class="govuk-heading-m vh-ml70 vh-mr70">Please wait while we process your request</h1>
-
+    <h1 class="govuk-heading-m vh-ml70 vh-mr70">Confirming bookings can take up to 60 seconds to complete.</h1>
```

Comment:
Again, I think this should be a translated string. I know you haven't put this in but it will likely be raised as a bug when the Welsh is tested

Reply:
This applies to all strings displayed to the user.

Reply:
as dicussed, Admin does not have language support